web/src/components/feature/FluidDialog.tsx

```diff
@@ -33,6 +60,6 @@ export const FluidDialog = ({
         </Button>
         <Button type="submit">Save</Button>
       </Dialog.Actions>
-    </Dialog>
+    </WideDialog>
```

Comment:
I think (some of) the buttons should be on the right side?

Reply:
Yes, but that is outside the scope of this task